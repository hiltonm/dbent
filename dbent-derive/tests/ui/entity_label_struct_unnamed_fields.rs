
use dbent::prelude::*;

#[derive(Label)]
struct Test(Key<Int>, #[label] String);

fn main() {}
