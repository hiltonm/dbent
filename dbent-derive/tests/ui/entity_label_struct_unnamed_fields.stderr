error: #[derive(Label)] can only be used on structs with named fields
 --> tests/ui/entity_label_struct_unnamed_fields.rs:5:12
  |
5 | struct Test(Key<Int>, #[label] String);
  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
